bundles/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@vaadin/bundles",
+  "version": "23.0.0-beta1",
```

Comment:
>"version": "23.0.0-beta1"

so we will need to automate this version? 
any other things with this file?

Reply:
i created a PR to update this version and also automate this package.json generation. https://github.com/vaadin/platform/pull/2738